packages/@react-spectrum/textfield/src/TextFieldBase.tsx

```diff
@@ -136,9 +147,12 @@ function TextFieldBase(props: TextFieldBaseProps, ref: Ref<TextFieldRef>) {
               styles,
               'spectrum-Textfield-input',
               {
-                'spectrum-Textfield-inputIcon': icon
+                'spectrum-Textfield-inputIcon': icon,
+                'spectrum-Textfield--resizableX': isResizeableWidth,
+                'spectrum-Textfield--resizableY': isResizeableHeight
               },
-              inputClassName
+              inputClassName,
+              classNames(formFieldStyles, {'spectrum-Form-input': isInForm})
```

Comment:
related to the above comments, maybe slap a `&& !isQuiet` here as well
```suggestion
              classNames(formFieldStyles, {'spectrum-Form-input': isInForm && !isQuiet})
```